src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -38,6 +39,9 @@ export default function EnterprisePage({ children, useEnterpriseConfigCache }) {
     return <NotFoundPage />;
   }
 
+  // push the optimizely user attributes here, where enterpriseConfig must be defined and not null
+  pushUserCustomerAttributes(enterpriseConfig);
```

Comment:
Wondering if this should be in a `useEffect` so it's only called upon component mount instead of any re-renders, too? Along the lines of:

```jsx
useEffect(() => {
  if (enterpriseConfig) { pushUserCustomerAttributes(enterpriseConfig); }
}, [enterpriseConfig]);
```

As is, I believe we will always be calling this twice even if the component never re-renders.

Note this `useEffect` would have to be added before any conditionals to avoid breaking the Rules of Hooks.